demos/text_detection_demo/main.cpp

```diff
@@ -137,12 +141,15 @@ int main(int argc, char *argv[]) {
         if (!FLAGS_m_tr.empty())
             text_recognition.Init(FLAGS_m_tr, ie, FLAGS_d_tr);
 
+        slog::info << "Reading input" << slog::endl;
         std::unique_ptr<Grabber> grabber = Grabber::make_grabber(FLAGS_dt, FLAGS_i);
         int wait_time = (FLAGS_dt == "image" || FLAGS_dt == "list") ? 0 : 3;
 
         cv::Mat image;
         grabber->GrabNextImage(&image);
 
+        slog::info << "Start inference " << slog::endl;
```

Comment:
This should probably be "Starting inference" for consistency with the other messages. Also, there's a trailing space.

Reply:
Doen